techs/js-borschik-include.js

```diff
@@ -1,32 +1,47 @@
+var EOL = require('os').EOL;
+
 /**
- * js-borschik-include
- * ===================
+ * @class JSBorschikIncludeTech
+ * @augments {BaseTech}
+ * @classdesc
+ *
+ * Collects js files declared by borschik (https://github.com/bem/borschik) includes.
+ * It must be used in case if you use borschik include declarations in your js files.
```

Comment:
It must be used in case if you use borschik include declarations in your js files. —> Use this technology to compile source files that contain borschik includes.